src/components/Loading.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component, PropTypes} from 'react';
```

Comment:
loading component (need to be added to every page)

![image](https://cloud.githubusercontent.com/assets/1591203/23692473/dfa315b6-039c-11e7-8e37-3f955bcca9d5.png)

the screenShot doesn't move but it is a gif :wink: